hackclub.com.yaml

```
@@ -2958,3 +2958,7 @@ shipment-viewer:
   - ttl: 600
     type: CNAME
     value: cname.vercel-dns.com.
+for-reviewers:
+   - ttl: 600
+    type: CNAME
+    value: souptik-samanta.github.io.
```

```suggestion
   - ttl: 600
     type: CNAME
     value: souptik-samanta.github.io.
```

just need to align the indentation

---

Done

---

hey just changed that